checkov/terraform/checks/resource/aws/DAXEncryption.py

```diff
@@ -0,0 +1,16 @@
+from checkov.terraform.checks.resource.base_resource_value_check import BaseResourceValueCheck
+from checkov.common.models.enums import CheckCategories
+
+
+class DAXEncryption(BaseResourceValueCheck):
+    def __init__(self):
+        name = "Ensure DAX is encrypted at rest (default is unencrypted)"
+        id = "CKV_AWS_47"
+        supported_resources = ['aws_dax_cluster']
+        categories = [CheckCategories.BACKUP_AND_RECOVERY]
```

Comment:
`CheckCategories.ENCRYPTION` might be a better fit here

Reply:
ctrl+c ctrl+v .... fixed it.